charts/argo-rollouts/values.yaml

```diff
@@ -25,7 +25,7 @@ controller:
   #    memory: 64Mi
 
   metrics:
-    enabled: false
+    enabled: true
```

Comment:
Is that change required? If so, please update the README :-)

Reply:
@mkilchhofer , it was enabled for test the PR. I reverted it back to `false` in the update PR. Thanks.